rules.bzl

```diff
@@ -28,6 +28,9 @@ def _android_ndk_repository_impl(ctx):
         fail("Either the ANDROID_NDK_HOME environment variable or the " +
              "path attribute of android_ndk_repository must be set.")
 
+    if ndk_path.startswith("$WORKSPACE_ROOT"):
+        ndk_path = str(ctx.workspace_root) + ndk_path[15:]
```

Comment:
for clarity let's make this `ndk_path.removeprefix("$WORKSPACE_ROOT")`

Reply:
Good point, done.